### PR TITLE
ARC-839 remove error pg FF and unused files for UI

### PR DESCRIPTION
**Pull request by rachellerathbone**
Last of the FF removals for the UI changes.

### CODE REVIEW

**Review by yvonnevictoria — APPROVED** [review]

File: views/maintenance.hbs [hunk 1]
```diff
@@ -30,7 +30,7 @@
 </head>
 <body class="aui-page-hybrid">
 <div class="ac-content">
-  <div class="page-content">
+  <div class="pageContent">
```
Comment (mboudreau) [inline views/maintenance.hbs:33]: Why this change?  Also, rule of thumb for css classes is to use kebab case, not camel case.

File: views/maintenance.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <title>{{title}}</title>
   <link rel="stylesheet" href="/public/css-reset/bundle.css" media="all" />
   <link rel="stylesheet" href="/public/atlassian-ui-kit/bundle.css" media="all" />
-  <link rel="stylesheet" href="/public/css/styles.css" media="all" />
+  <link rel="stylesheet" href="/public/css/global.css" media="all" />
```
Comment (mboudreau) [inline views/maintenance.hbs:11]: was this meant to be?

File: views/maintenance.hbs (hunk 1 above)
Comment (rachellerathbone) [reply views/maintenance.hbs:33]: As part of the UI updates everything that was shared went into `global.css`. With plain CSS, yes, but when using BEM conventions, you can choose to use camel case. https://en.bem.info/methodology/naming-convention/

File: views/maintenance.hbs (hunk 2 above)
Comment (rachellerathbone) [reply views/maintenance.hbs:11]: Yep. As part of the UI changes I renamed styles as a file with globally shared styling, it makes more sense for the file to be named 'global'.

File: views/maintenance.hbs (hunk 1 above)
Comment (KAbakumovAtl) [reply views/maintenance.hbs:33]: Is it how it named in global.css? If not, should we rename it in the css file as well? 🤔

File: views/maintenance.hbs (hunk 2 above)
Comment (mboudreau) [reply views/maintenance.hbs:11]: agreed :)

File: views/maintenance.hbs (hunk 1 above)
Comment (mboudreau) [reply views/maintenance.hbs:33]: It's fine for now, I think we'll need to discuss in the future so that we use an agreed upon standard across the board.

**Review by mboudreau — APPROVED** [review]
